index.html

```diff
@@ -0,0 +1,236 @@
+                                <p class="m2">Lorem ipsum dolor sit amet consectetur adipisicing elit. Nemo autem ducimus enim? Veritatis adipisci error nostrum. Unde dicta libero</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="section-4 bg-dark">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-7">
+                        <img src="images/join.jpg" alt="Image5" class="img-fluid" width="590">
+                    </div>
+                    <div class="col-md-5">
+                        <h1 class="text-white">
+                            Don't Know Where to start From?
+                        </h1>
+                        <a href="#">Join Us</a>
+
+                        <p class="para-1">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Velit, architecto eveniet natus veniam qui rem modi, quo consequatur suscipit quibusdam ipsam iste debitis, iure ducimus enim laboriosam blanditiis nobis quis.</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </main>
+
+
+    <footer>
+        <div class="section-5 text-center">
+            <h4 style="margin-top: 5%">Take a course from ITcourses</h4>
+            <h4 class="my-4">If you need any help</h4>
+
+            <div class="col-md-4 form-inline m-auto">
+                <input type="text" name="Email" id="email" placeholder="Email" size="40" class="form-control px-4 py-2">
+                <input type="button" value="Contact Us" class="btn btn-danger px-4 py-2 mt-3">
+            </div>
+
+            <div class="social col-md-4 m-auto mt-5">
+                <div class="d-flex flex-row justify-content-center">
+                    <a href="#"><i class="fab fa-facebook-f m-2"></i></a>
+                    <a href="#"><i class="fab fa-twitter m-2"></i></a>
+                    <a href="#"><i class="fab fa-instagram m-2"></i></a>
+                    <a href="#"><i class="fab fa-youtube m-2"></i></a>
+                </div>
+            </div>
+
+            <hr>
+            <h5 style="color:lightseagreen" class="mb-3">ITcourses &copy;</h5>
+        </div>
+    </footer>
+
+    
+
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.6.0/dist/umd/popper.min.js" integrity="sha384-KsvD1yqQ1/1+IA7gi3P0tyJcT3vR+NdBTt13hSJ2lnve8agRGXTTyNaBYmCR/Nwi" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.min.js" integrity="sha384-nsg8ua9HAw1y0W1btsyWgBklPnCUAFLuTMS2G72MMONqmOymq585AcH49TLBQObG" crossorigin="anonymous"></script>
```

Comment:
As described [in the doc](https://getbootstrap.com/docs/5.0/getting-started/introduction/#js) you had to include only `bootstrap.bundle.min.js` script or `Popper` and `bootstrap.min.js`.

```suggestion
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
```